nsiqcppstyle_exe.py

```diff
@@ -78,91 +77,138 @@ def Usage():
                 to create one (e.g., in a read-only file system)
   --var=key: value,key: value
                 provide the variables to customize the rule behavior.
-  --list-rules / -r  Show all rules available.
-                Add file extensions to be counted as assigned languages.
-  -s            Assign Filter scope name to be applied in this analysis
-  --output=     output format 'emacs', 'vs7', 'csv', 'xml' and 'eclipse'. Default value is vs7
-                emacs, vs7, eclipse output the result on the stdout in the form
-                that each tool recognizes.
-                csv and xml outputs the result on the file "nsiqcppstyle_result.csv"
-                "nsiqcppstyle_result.xml" respectively, if you don't provide -o option.
-  --ci          Continuous Integration mode. If this mode is on, this tool only reports summary.
-  --quiet / -q  Quiet mode. If this mode is on, this tool only reports errors.
+
 
 * nsiqcppstyle reports coding standard violations on C/C++ source code.
-* In default, it doesn't apply any rules on the source. If you want to apply rule,
-  they should be provided in the filefilter.txt file in following form.
+
+* By default, it doesn't apply any rules on the source. If you want to apply rule, they should be provided in the 'filefilter.txt' file in following form.
   ~ RULENAME
 
-* You can customize the rule behavior by providing --var=key: value pair when executing
-  tool and you can put it in the filefilter.txt. The format is following.
+* You can customize the rule behavior by providing --var=key: value pair when executing tool and you can put it in the 'filefilter.txt'. The format is following.
```

Comment:
Grammar: "It can be inserted in the 'filefilter.txt' file using the following format:"

Reply:
```suggestion
* You can customize the rule behavior by providing --var=key: value pair when executing the tool. You can also insert them in the 'filefilter.txt' with the following format:
```